### PR TITLE
[Magic Network Monitoring] Added new tile

### PR DESCRIPTION
**Pull request by dcpena**
Added new tile for Magic Network Monitoring to address PCX-4271.

### CODE REVIEW

File: content/magic-network-monitoring/get-started.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+title: Get started
+pcx_content_type: overview
+weight: 2
+---
+
+# Get started
+
+To begin using Magic Network Monitoring, complete the list of tasks below. If you are an Enterprise customer, Cloudflare can significantly accelerate the timeline during active-attack scenarios.
+
+If you are an Enterprise customer and would like to use Magic Network Monitoring and Magic Transit On Demand together, begin by [configurin gMagic Transit](/magic-transit/get-started/).
```
Comment (kodster28) [inline content/magic-network-monitoring/get-started.md]:
```suggestion
If you are an Enterprise customer and would like to use Magic Network Monitoring and Magic Transit On Demand together, begin by [configuring Magic Transit](/magic-transit/get-started/).
```

File: content/magic-network-monitoring/_index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Detect and receive notifications about attacks based on traffic flows
+
+{{</description>}}
+
+{{<plan type="all">}}
+
+Magic Network Monitoring provides visibility into your network traffic by analyzing network flow data sent from a customer’s routers. Magic Network Monitoring supports NetFlow v5, NetFlow v9, IPFIX, and sFlow.
+
+Enterprise customers can use Magic Network Monitoring with [Magic Transit on-demand](https://developers.cloudflare.com/magic-transit/on-demand/) to monitor their network, identify volumetric DDoS attacks, and activate Magic Transit on-demand to mitigate those attacks.
```
Comment (kodster28) [inline content/magic-network-monitoring/_index.md]:
```suggestion
Enterprise customers can use Magic Network Monitoring with [Magic Transit on-demand](/magic-transit/on-demand/) to monitor their network, identify volumetric DDoS attacks, and activate Magic Transit on-demand to mitigate those attacks.
```

File: content/magic-network-monitoring/get-started.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Refer to [Supported routers](/magic-network-monitoring/routers/supported-routers) to view a list of supported routers. The list is not exhaustive. 
+
+Magic Network Monitoring supports NetFlow v5, NetFlow v9, IPFIX, and sFlow.
+
+## 2. Register your router with Cloudflare
+
+When you register your router with Cloudflare, your router links your NetFlow or sFlow data to your Cloudflare account.
+
+1. From the Cloudflare dashboard, select **Magic Network Monitoring**.
```
Comment (kodster28) [inline content/magic-network-monitoring/get-started.md]: I'd add a dash link here.

File: content/magic-network-monitoring/magic-transit-integration.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+weight: 6
+---
+
+#  Magic Transit integration
+
+[Magic Transit On Demand](/magic-transit/on-demand/) customers need to know when they should activate Magic Transit to mitigate DDoS attacks, and Magic Network Monitoring solves this problem by analyzing traffic flow from your data centers and detecting DDoS attacks.
+
+If an attack is detected, you can choose to automatically activate IP advertisement and enable Magic Transit to protect the impacted IP prefixes from DDoS attacks. This feature is refer to as auto-advertisement, and you can enable it for individual Magic Network Monitoring rules. After Magic Transit is activated and your traffic is flowing through Cloudflare, you only receive the clean traffic back to your network over your tunnels.
+
+To activate IP advertisement via the Cloudflare dashboard, refer to [​using the IP Prefixes page to configure dynamic advertisement](https://developers.cloudflare.com/byoip/how-to/configure-dynamic-advertisement/#configure-dynamic-advertisement-via-the-dashboard).
```
Comment (kodster28) [inline content/magic-network-monitoring/magic-transit-integration.md]:
```suggestion
To activate IP advertisement via the Cloudflare dashboard, refer to [​using the IP Prefixes page to configure dynamic advertisement](/byoip/how-to/configure-dynamic-advertisement/#configure-dynamic-advertisement-via-the-dashboard). 
```

File: content/magic-network-monitoring/routers/supported-routers.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+- [pfSense](https://www.pfsense.org/) (open source)
+  - [pfSense NetFlow package](https://docs.netgate.com/pfsense/en/latest/recipes/netflow-with-softflowd.html)
+- [OpenWrt](https://openwrt.org/start) (open source)
+  - [Table of supported routers](https://openwrt.org/toh/start)
+  - [OpenWrt NetFlow support](https://openwrt.org/packages/pkgdata/softflowd)
+-  [Ubiquiti Edge Routers](https://store.ui.com/collections/operator-edgemax-routers)
+
+## NetFlow Routers
+
+- Aruba Networks
```
Comment (kodster28) [inline content/magic-network-monitoring/routers/supported-routers.md]: Probably not understanding this right, but should we link to our other docs on Aruba?

File: data/magic-network-monitoring.yml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  url: /magic-network-monitoring/
+
+meta:
+  title: Cloudflare Magic Network Monitoring docs
+  description: Magic Network Monitoring provides visibility into your network traffic by analyzing network flow data sent from a customer's routers.
+  author: '@cloudflare'
+  image: /cf-twitter-card.png
+
+resources:
+  dashboard_link:
```
Comment (kodster28) [inline data/magic-network-monitoring.yml]: Did you mean to fill this in? I'd think so since it sounds like it has a specific area in the dash?

**Review by kodster28 — APPROVED** [review]

File: data/magic-network-monitoring.yml (hunk 6 above)
Comment (dcpena) [reply data/magic-network-monitoring.yml]: Ah, good catch! Added

File: content/magic-network-monitoring/routers/supported-routers.md (hunk 5 above)
Comment (dcpena) [reply content/magic-network-monitoring/routers/supported-routers.md]: Nah, this is something different than what the Aruba tutorial is covering.

File: content/magic-network-monitoring/get-started.md (hunk 3 above)
Comment (dcpena) [reply content/magic-network-monitoring/get-started.md]: done!

**Pull request merged**